metrics/group_processes.go

```diff
@@ -0,0 +1,94 @@
+// Copyright 2022 Tigris Data, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package metrics
+
+import (
+	"github.com/rs/zerolog/log"
+	"github.com/tigrisdata/fdb-exporter/models"
+)
+
+type ProcessesMetricGroup struct {
+	metricGroup
+}
+
+func NewProcessesMetricGroup(reporter *MetricReporter) *ProcessesMetricGroup {
+	return &ProcessesMetricGroup{*newMetricGroup("processes", reporter.GetScopeOrExit("cluster"), reporter)}
+}
+
+func (p *ProcessesMetricGroup) getTags(processName string, process *models.Process) map[string]string {
```

Comment:
It might make the code more readable if we can make that `map[string]string` and type of some sorts on the longer run. Not a blocker though.

Reply:
Right, than we need to convert it back to tally, and the conversion back and forth has a cost. Let me think about this.